docs/reference/configuration.txt

```diff
@@ -369,7 +369,9 @@ for details on driver options.
 
       # Maintain legacy behavior of pluck and distinct, which does not demongoize
       # values on returning them. Setting this option to false will cause
-      # pluck and distinct to return demongoized values.
+      # pluck and distinct to return demongoized values. This will also allow
+      # passing _translations fields to pluck and distinct and will return
+      # embedded values themselves (i.e. without putting them in a hash).
```

Comment:
The blurb you're adding is for the false value of the option, whereas the existing description is for the true value.

Reply:
Ah I figured saying "This will also allow" was referring to the previous sentence which was talking about setting it to false. I can make that more clear.

Reply:
done